tasks/e2e.sh

```diff
@@ -135,7 +138,7 @@ npm install $cli_path
 
 # Install the app in a temporary location
 cd $temp_app_path
-create_react_app --scripts-version=$scripts_path test-app
+create_react_app --scripts-version=$scripts_path --template=$root_path/packages/react-scripts/fixtures/kitchensink test-app
```

Comment:
After removing some tests from the kitchensink version (e.g. the `--scripts-version` ones) I'm quite satisfied with the timing.
The main fear I have is that sometimes it could exceed the travis 1hour limit..